catalog/systemd.pl.catalog.in

```diff
@@ -353,3 +353,22 @@ Defined-By: systemd
 Support: %SUPPORT_URL%
 
 Jednostka @UNIT@ została ukończona, zużywając wskazane zasoby.
+
+-- 50876a9db00f4c40bde1a2ad381c3a1b
+Subject: System jest skonfigurowany w sposób, który może powodować problemy
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Możliwe są następujące „etykiety”:
+• „split-usr” — /usr jest oddzielnym systemem plików, który nie został
+  zamontowany podczas uruchamiania systemd,
```

Comment:
This needs to be in a different tense, maybe "który nie był zamontowany w momencie uruchomienia systemd".

Reply:
Done (as “który nie był zamontowany w czasie uruchomienia systemd”).